src/Components/test/testassets/BasicTestApp/RouterTest/LocationChangingCancel.razor

```diff
@@ -0,0 +1,46 @@
+﻿@page "/LocationChangingCancel"
+
+@using Microsoft.JSInterop
+@using Microsoft.AspNetCore.Components.Routing
+
+@inject IJSRuntime JSRuntime
+@inject NavigationManager NavigationManager
+@implements IDisposable
+
+<button id="test-button" @onclick=@( () => NavigationManager.NavigateTo("LocationChangingStart"))>Navigate</button>
+
+<a id="test-link" href="LocationChangingStart">Can We</a>>
```

Comment:
Can we add another test scenario for a link to an external site here (e.g. "google.com")?

Reply:
I will add that test

Reply:
Test have been added